tracing-opentelemetry/src/main/java/com/couchbase/client/tracing/opentelemetry/OpenTelemetryRequestSpan.java

```diff
@@ -78,9 +78,7 @@ public void addEvent(String name, Instant timestamp) {
 
   @Override
   public void end() {
-    try (Scope scope = span.makeCurrent()) {
```

Comment:
So if the span  is completed on a  different  thread than it  was created, it doesn't need to made current again?

Reply:
Not just to call `.end`

If in this method, we had a call like `grpcClient.executeRpc`, then yes it would need to be made current so the gRPC client picked it up as a parent. Making a span current is only to allow implicit propagation between two different libraries.

Reply:
Ok that makes sense, thanks